packages/aws-cdk-lib/aws-ec2/test/vpn.test.ts

```diff
@@ -212,6 +212,28 @@ describe('vpn', () => {
 
   });
 
+  test('does not fail when specifying two tunnels but no tunnelInsideCidr', () => {
+    // GIVEN
+    const stack = new Stack();
+
+    expect(() => new Vpc(stack, 'VpcNetwork', {
+      vpnConnections: {
+        VpnConnection: {
+          ip: '192.0.2.1',
+          tunnelOptions: [
+            {
+              preSharedKeySecret: SecretValue.unsafePlainText('secretkey1234'),
+            },
+            {
+              preSharedKeySecret: SecretValue.unsafePlainText('secretkey5678'),
+            },
+          ],
+        },
+      },
+    })).not.toThrow();
+
+  });
+
```

Comment:
```suggestion
  test('with two tunnel options and no tunnelInsideCidr', () => {
    // GIVEN
    const stack = new Stack();

    // WHEN
    new Vpc(stack, 'VpcNetwork', {
      vpnConnections: {
        VpnConnection: {
          ip: '192.0.2.1',
          tunnelOptions: [
            {
              preSharedKeySecret: SecretValue.unsafePlainText('secretkey1234'),
            },
            {
              preSharedKeySecret: SecretValue.unsafePlainText('secretkey5678'),
            },
          ],
        },
      },
    });

    // THEN
    Template.fromStack(stack).hasResourceProperties('AWS::EC2::VPNConnection', {
      CustomerGatewayId: {
        Ref: 'VpcNetworkVpnConnectionCustomerGateway8B56D9AF',
      },
      Type: 'ipsec.1',
      VpnGatewayId: {
        Ref: 'VpcNetworkVpnGateway501295FA',
      },
      StaticRoutesOnly: false,
      VpnTunnelOptionsSpecifications: [
        {
          PreSharedKey: 'secretkey1234',
        },
        {
          PreSharedKey: 'secretkey5678',
        },
      ],
    });
  });

```
Let's check also the template for better coverage.